diff --git a/assets/lyrics.js b/assets/lyrics.js
index 77ac9d3..ebdab83 100644
--- a/assets/lyrics.js
+++ b/assets/lyrics.js
@@ -1274,18 +1274,10 @@ function initialiseNotesEditor()
                         "SAVE RESULT:",
                         result
                     );
-                                        
+                      
+
             if (result.success)
                 {
-                    
-                    userNotes = notes;
-                    
-                    clearUserNotesDisplay();
-                    
-                    insertUserNotes(userNotes.user, userNotes.inline);
-                    insertTopUserNotes(userNotes.user, userNotes.songNotes);
-                    insertSectionUserNotes(userNotes.user, userNotes.sections);
-
                     closeNotesEditor();
                 }
 
@@ -1520,3 +1512,12 @@ for (const row of lineRows)
     return notes;
 }
 
+async function refreshUserNotes()
+{
+    clearUserNotesDisplay();
+
+    userNotes = null;
+
+    await loadUserNotes();
+}
+
